cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineInstanceDTO.java

```diff
@@ -39,6 +40,8 @@
 
     private List<ApplicationDTO> installation;
 
+    private OsFamily osFamily;
```

Comment:
operatingSystemFamily

Reply:
lo esponiamo già in `InstancePricingDTO` ed è `osFamily`